adafruit_platformdetect/chip.py

```diff
@@ -121,6 +121,10 @@ def _linux_id(self):  # pylint: disable=too-many-branches,too-many-statements
                 linux_id = chips.A64
             elif "sun50iw1p1" in hardware:
                 linux_id = chips.A64
+            elif "Xilinx Zynq" in hardware:
```

Comment:
Shouldn't we compare either in lower or capital letters?

Reply:
This matches the cpuinfo output from the kernel. We could make the matching case insensitive, but as the kernel always returns this in a consistent way, I'm not sure what the point would be.

Reply:
Would be the same in all distributions? Just asking cuz it happened to me with the Pinephone that the info was in different formats like `PinePhone, pinephone`.

Also I don't have the last word here to approve the PR :( I guess Melissa will see if this is necessary or not.

Reply:
In the PYNQ case, yes, as we read it from procfs directly, which means that the kernel is responsible for the formatting, and this will be consistent regardless. In your pinephone case, you are reading the model name from the device tree, which somewhat puts you at the mercy of the people preparing the device tree blob. The convention there is all lowercase, but it wouldn't surprise me if people left in mixed-case compat strings, too.